packages/client/e2e/_utils/page.ts

```diff
@@ -97,6 +103,21 @@ export async function initPage(page: Page): Promise<void> {
     await focusTrap.click();
 }
 
+export async function PerformSignUp(page: Page): Promise<void> {
+    const signUpButton = page
+        .locator(`css=[data-testid="sign-up-button"]`)
+        .last();
+    await expect(signUpButton).toBeVisible();
+
+    await signUpButton.click();
+
+    await expect(
+        page.locator(`text="Signed up successfully"`).last(),
+    ).toBeVisible();
+
+    await page.reload();
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I like that!